lib/net.js

```diff
@@ -1563,12 +1563,12 @@ exports.isIP = cares.isIP;
 
 
 exports.isIPv4 = function(input) {
-  return exports.isIP(input) === 4;
+  return cares.isIPv4(input);
```

Comment:
Is there a reason why you don't just assign it as `exports.isIPv4 = cares.isIPv4;` ?